sys-apps/mlocate/files/updatedb.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Update locate database
+
+[Service]
```

Comment:
The current cron job doesn't run *updatedb* when on battery. Please consider adding `ConditionACPower=true` or tell us why you don't want this condition for systemd users.

The current cron job takes care of "NONDEVS" to avoid errors. Consider adding a Condition on the existence of */etc/updatedb.conf* file.

The current cron job cleans up old temp files in */var/lib/mlocate/* from previous aborted runs. The systemd service should do the same to avoid things like https://askubuntu.com/questions/117162/how-do-i-reduce-the-size-of-mlocate-database